pkg/kubelet/prober/prober.go

```diff
@@ -192,10 +192,10 @@ func (pb *prober) runProbe(probeType probeType, p *v1.Probe, pod *v1.Pod, status
 		if host == "" {
 			host = status.PodIP
 		}
-		klog.V(4).Infof("TCP-Probe Host: %v, Port: %v, Timeout: %v", host, port, timeout)
+		klog.V(4).InfoS("TCP-Probe Host", "host", host, "port", port, "timeout", timeout)
 		return pb.tcp.Probe(host, port, timeout)
 	}
-	klog.Warningf("Failed to find probe builder for container: %v", container)
+	klog.InfoS("Failed to find probe builder for container", "container", container.Name)
```

Comment:
```suggestion
	klog.InfoS("Failed to find probe builder for container", "containerName", container.Name)
```

Reply:
Found a bunch of others in the file, fixed those too.